### ISSUE TITLE
Page doesn't load when it includes an undefined link

### ISSUE THREAD

**Issue by lisacharlotterost** [interactivethings/catalog#435]
Thanks for the great framework! One problem: Our pages don't seem to load when we put undefined links in there (e.g. `hello@hello.de`). I get the following errors: 
```
Warning: Failed prop type: The prop `to` is marked as required in `Link$$1`, but its value is `undefined`.
Uncaught TypeError: Cannot read property 'indexOf' of undefined
The above error occurred in the <Link$$1> component
Uncaught (in promise) TypeError: Cannot read property 'indexOf' of undefined
```

Yes, it works as soon as I define the link `[hello@hello.de](mailto:hello@hello.de)`, but should users be forced to define their links? And if so, you might want to give them a better error message :) Thanks!

**Comment by herrstucki** [comment]
Thanks for reporting this! It’s definitely a bug!

And I agree that we need nicer error messages 😅

**Comment by herrstucki** [comment]
@lisacharlotterost fix published in v3.5.5 ✌️
